DigitalLearningSolutions.Web.Tests/Helpers/FilteringHelperTests.cs

```diff
@@ -32,7 +32,21 @@ public void FilterItems_returns_expected_items_with_multiple_filters()
         {
             // Given
             var expectedItems = new[] { ItemA1 }.AsQueryable();
-            var filterBy = "Name|a\r\nNumber|1";
+            var filterBy = "Name|Name|a\r\nNumber|Number|1";
+
+            // When
+            var result = FilteringHelper.FilterItems(InputItems, filterBy);
+
+            // Then
+            result.Should().BeEquivalentTo(expectedItems);
+        }
+
+        [Test]
+        public void FilterItems_with_filters_in_same_category_returns_union_of_filters()

```

Comment:
I think we should have a few extra tests for this functionality:
* Filtering two different categories not taking union
* Filtering twice per category not returning duplicates if one item matches both filters in that category
* Case where we test it all together - e.g. two filters in same category and one in a different category

Reply:
Filters in 2 categories already had a test FilterItems_returns_expected_items_with_multiple_filters